cmd/image-builder-db-test/main_test.go

```diff
@@ -112,9 +112,23 @@ func testMigration(t *testing.T) {
 
 	// Verify that after migration step 3 adding a compose request to the db requires a non empty account number.
 	d, err := db.InitDBConnectionPool(connStr(t))
-	err = d.InsertCompose(uuid.New().String(), "", ORGID1, []byte("{}"))
+	err = d.InsertCompose(uuid.New().String(), "", ORGID1, nil, []byte("{}"))
```

Comment:
Because this would be using current defined Insertcomposer against a migration-3 db, it needs to be transformed into a direct query:

```
insert = "INSERT INTO composes(job_id, request, created_at, account_id, org_id) VALUES ($1, $2, CURRENT_TIMESTAMP, $3, $4)"
_, err = conn.Exec(context.Background(), insert, uuid.New().String(), "{}", "", ORGID1)
require.Error(t, err)
```

Sorry should've mentioned that. Last thing and it should be good :)

Reply:
ahh makes sense.